### ISSUE TITLE
Don't register click on already answered area

### ISSUE THREAD

**Issue by SmokeysDev** [codeforamerica/click_that_hood#311]
Long version:
Couldn't spot an existing issue on this, but might have come up before. Since we're playing quickly to get fast times, and some countries/areas are small, I subconsciously allow myself more margin for error if there's one unused region surrounded by green (already completed) ones, but it registers the misclick just outside of my intended region as an incorrect answer (and considering the status of the region I ended up clicking, it should be fairly obvious that I didn't think it was that one).

Short version:
Request - Closely missing an incomplete region and clicking a green/complete region - ignore the click, rather than count it as an error

Obviously a close miss into another possible region can't be handled, but would be nice to have some leeway with the already answered ones. Hope this makes sense!

**Comment by SmokeysDev** [comment]
Totally forgot I was a dev for a minute... As it's such a simple one, figured I'd just code it and create a PR #313 - hopefully less hassle then, if the feature is agreed
